### ISSUE TITLE
Various music issues

### ISSUE THREAD

**Issue by serin-delaunay** [Attnam/ivan#187]
* The MIDI tempo slows down when I use the 'g'o command, or walk around the dungeon, or use other applications. This may be due to limitations of my computer (1.75 GHz, 64-bit single-core).
* When a badly damaged limb is severed, if the remaining bodyparts have full health, The music intensity drops to 0. This seems wrong - is there any way to account for limb absence in the intensity calculation?
* Dungeon2.mid doesn't loop smoothly.
* If I use troll blood to regrow a full-health limb, the intensity goes to maximum. After I take a turn, the intensity decreases back to the correct level. Similar problems (but slightly less pronounced?) seem to occur when simply healing damage with a potion.
* Often when transitioning between maps, especially from a location with music to the world map, the last note played before the transition suddenly increases in volume.

**Comment by AquariusPower** [comment]
On linux, midi music never works initially. 
I have to restart the game many times before it will work but sometimes I just giveup. 

if I play the midi from command line using `aplaymidi` (alsa), it will always work well.
So I guess, there is some problem on the midi player of the game.

PS.: To control fluidsynth (that enables midi music) I use QSynth.

**Comment by fejoa** [comment]
@AquariusPower Did you change the Midi output device in the configuration menu?

**Comment by AquariusPower** [comment]
yes I did, 
but after more tests, I now believe the problem is actually in fluidsynth/QSynth and not on ivan.
As it is working quite well after many ivan restarts, and every time is working (today at least).

what I stopped doing was clicking the panic button on QSynth whenever anything weird happens to the music (like volume lowered, may be interference of pulseaudio ladspa plugin to normalize audio volume).

also, if I start QSynth after ivan it would bug, I wont even try now... may break again.
or if the machine is under heavy load ( like writing to pendrive LOL :/ ), QSynth may bug out too.

In short, may be there is nothing that can be done on ivan, 
it actually seems a OS/QSynth issue.

**Comment by AquariusPower** [comment]
on linux at least, 
I've found that, when music changes, the instruments are not updated, what is fun but some musics get a bit weird sometimes, so I saved one preset for each music and I change it til it matches the good instruments :).
I think the in-game music changing "reset?" step has something to do with this problem.
